.golangci.yml

```diff
@@ -111,5 +111,8 @@ issues:
     - path: controlplane/pkg/monitor/
       linters:
         - dupl
+    - path: controlplane/pkg/nsm.nsm.go
+      linters:
+        - gocyclo # TODO: reduce nsm.request() complexity
```

Comment:
Can you create an issue for this problem?

Reply:
Here it is: #1241.